apis/nucleus/src/components/listbox/assets/horizontal-minimum-batch-size.js

```diff
@@ -0,0 +1,5 @@
+export default function getHorizontalMinBatchSize({ width, columnWidth, listHeight, itemSize }) {
+  const visibleCellsCount = Math.ceil(width / columnWidth) * Math.ceil(listHeight / itemSize);
+  const minSize = visibleCellsCount * 2;
```

Comment:
```suggestion
  return visibleCellsCount * 2;
```

Reply:
I will keep it mainly for debugging purpose.